app/.s2i/bin/assemble

```diff
@@ -51,6 +51,10 @@ elif [[ -f setup.py ]]; then
 fi
 
 cd frontend
+npm cache clean -f
+npm install n
+source ~/.bashrc
+n 8.11.3
```

Comment:
What us n ?

Reply:
It's a node version manager.  Opinion?
https://www.npmjs.com/package/n